cli/src/main/java/com/liferay/blade/cli/util/ListUtil.java

```diff
@@ -33,4 +51,56 @@
 		return new ArrayList<>(Arrays.asList(array));
 	}
 
+	public static boolean isEmpty(Collection<?> collection) {
+		if ((collection == null) || collection.isEmpty()) {
+			return true;
+		}
+
+		return false;
+	}
+
+	public static boolean isEmpty(List<?> list) {
+		if ((list == null) || list.isEmpty()) {
+			return true;
+		}
+
+		return false;
+	}
+
+	public static boolean isEmpty(Object[] array) {
+		if ((array == null) || (array.length == 0)) {
+			return true;
+		}
+
+		return false;
+	}
+
+	public static boolean isEmpty(Set<?> set) {
+		if ((set == null) || set.isEmpty()) {
+			return true;
+		}
+
+		return false;
+	}
+
+	public static boolean isNotEmpty(Collection<?> collection) {
+		return !isEmpty(collection);
+	}
+
+	public static boolean isNotEmpty(List<?> list) {
```

Comment:
We should not need this method when we have `isNotEmpty(Collection<?> collection)`